pkg/util/flavor/flavor.go

```diff
@@ -22,6 +22,8 @@ const (
 	ServerlessAgent = "serverless_agent"
 	// HerokuAgent is the Heroku Agent flavor
 	HerokuAgent = "heroku_agent"
+	// ProcessAgent is the Process Agent
```

Comment:
Nit: looks like most comments here are in the form of `FooAgent is the Foo Agent flavor`
```suggestion
	// ProcessAgent is the Process Agent flavor
```

Reply:
Ah fair enough!